src/niveristand/clientapi/realtimesequence.py

```diff
@@ -123,10 +123,10 @@ def _transform(self):
         rtsequtils.compile_rtseq(self._rtseq)
 
     def _update_references(self):
-        self._ref = Reference(str(self), self._build_file_name())
+        self._ref.append(Reference(str(self), self._build_file_name()))
         self._rtseq.References = References()
         for referenced in self._rtseqpkg.get_referenced(self):
-            self._rtseq.References.AddReference(referenced.get_reference())
+            self._rtseq.References.AddReference(referenced.get_reference().pop())
```

Comment:
Don't you need all the references?

Reply:
No, you add the last saved reference and then you delete it, so the next time you want to add a reference to that RealTimeSequence, it is not already used